src/components/side_panel/pivot/pivot_layout_configurator/pivot_layout_configurator.xml

```diff
@@ -65,12 +65,19 @@
           t-on-pointerdown="(ev) => this.startDragAndDropMeasures(measure, ev)"
           t-att-style="dragAndDrop.itemsStyle[measure.id]"
           class="pt-1 pivot-measure">
-          <PivotDimension dimension="measure" onRemoved.bind="removeMeasureDimension">
-            <div class="d-flex flex-row small">
-              <div class="d-flex flex-row py-1 px-2 w-100">
+          <PivotDimension
+            dimension="measure"
+            onRemoved.bind="removeMeasureDimension"
+            onNameUpdated.bind="onMeasureNameUpdated">
+            <div class="d-flex flex-column small">
+              <div class="d-flex py-1 px-2 w-100">
+                <div class="pivot-dim-operator-label">Field</div>
```

Comment:
what do you think of displaying the field here only if the measure name is different from the field ? To have a cleaner side panel by default

Reply:
Actually, do we need to display the technical field name at all ?
We never displayed the technical field name in odoo, and it looks like it never bothered anyone. (as long as the autocomplete shows the user defined name + technical name)

Reply:
or maybe as a `title` tooltip?

Reply:
I would go for the field display name, the tooltip is useless I think. (And it's an override in odoo for nothing...)